adding tutorial for sending an SMS with asp.net (#3254)
Nexmo/nexmo-developer: _tutorials/en/sms-api/create-send-sms-view.md

---
title: Create a Send SMS View
description: Add an MVC View to
---

# Add a Send SMS View

> Add a new folder under `Views` called `Sms`, in there add a file `index.cshtml` and add the following code to it:

Reviewer: ```suggestion
Create a new folder under `Views` called `Sms`. Within `Sms`, add a file called `index.cshtml` that contains the following code:
```

Author: done
